geocatbridge/servers/models/geonetwork.py

```diff
@@ -141,6 +147,9 @@ def publishMetadata(self, metadata):
         with open(metadata, "rb") as f:
             files = {
                 'uuidProcessing': (None, 'OVERWRITE', 'text/plain'),
+                'group': (None, self.group, 'text/plain'),
+                'assignToCatalog': (None, self.assignCat, 'text/plain'),
+                'transformWith': (None, self.transform, 'text/plain'),
```

Comment:
@ByronCinNZ does this work properly when one or more of these values are an empty string?

Reply:
Yes it works fine